<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Computational Biology Lab @ CityU HK</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">News</a>
</li>
<li>
  <a href="people.html">People</a>
</li>
<li>
  <a href="publication.html">Publications</a>
</li>
<li>
  <a href="software.html">Software</a>
</li>
<li>
  <a href="collaboration.html">Collaborations</a>
</li>
<li>
  <a href="opening.html">Openings</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<div id="selected-publications" class="section level3">
<h3>Selected Publications</h3>
<p>(† co-first author; * co-corresponding author)</p>
<div id="section" class="section level4">
<h4>2018</h4>
<ol style="list-style-type: decimal">
<li>Kandimalla R†, <U>Gao F†</U>, Matsuyama T, Ishikawa T, Uetake H, Takahashi N, Yamada Y, Becerra C, Kopetz S, <strong>Wang X*</strong> and Goel A*. Genome-wide discovery and identification of a novel miRNA signature for recurrence prediction in stage II and III colorectal cancer. <strong>Clin Cancer Research</strong> 2018, <a href="doi:10.1158/1078-0432.CCR-17-3236" class="uri">doi:10.1158/1078-0432.CCR-17-3236</a><br />
</li>
<li>Xu Z, <U>Wang W</U>, Ren Y, Zhang W, Fang P, Huang L, <strong>Wang X</strong>, Shi P. Regeneration of cortical tissue from brain injury by implantation of defined molecular gradient of semaphorin 3A. <strong>Biomaterials</strong> 2018;157: 125–135.</li>
</ol>
</div>
<div id="section-1" class="section level4">
<h4>2017</h4>
<ol style="list-style-type: decimal">
<li><U>Qi L</U>, <U>Wang W</U>, Xing X, Wang K and <strong>Wang X</strong>, Complete canonical correlation analysis for multi-omic molecular subtyping of colorectal cancer, <strong>International Conference on Biological Information and Biomedical Engineering (BIBE 2018)</strong>, in press</li>
<li>Linnekamp JF†, van Hooff SR†, Prasetyanti PR, Kandimalla R, Buikhuisen JY, Fessler E, Ramesh P, Lee KAST, van Bochove GGW, de Jong JH, Cameron K, van Leersum R, Rodermond HM, Franitza M, Nürnberg P, Mangiapane LR, <strong>Wang X</strong>, Clevers H, Vermeulen L, Stassi G and Medema JP, Consensus Molecular Subtypes of colorectal cancer are recapitulated in in vitro and in vivo models, <strong>Cell Death and Differentiation</strong> 2018, <a href="doi:10.1038/s41418-017-0011-5" class="uri">doi:10.1038/s41418-017-0011-5</a><br />
</li>
<li>Sonohara F†, <U>Gao F†</U>, Iwata N, Kanda M, Koike M, Takahashi N, Yamada Y, Kodera Y, <strong>Wang X*</strong>, Goel A*, Genome-wide Discovery of a Novel Gene-expression Signature for the Identification of Lymph Node Metastasis in Esophageal Squamous Cell Carcinoma, <strong>Annals of Surgery</strong> 2017, <a href="doi:10.1097/SLA.0000000000002622" class="uri">doi:10.1097/SLA.0000000000002622</a><br />
</li>
<li>Ozawa T†, Kandimalla R†, <U>Gao F†</U>, Nozawa H, Hata K, Nagata H, Okada S, Izumi D, Baba H, Fleshman J, <strong>Wang X</strong>, Watanabe T, Goel A, A microRNA Signature Associated With Metastasis of T1 Colorectal Tumors to Lymph Nodes, <strong>Gastroenterology</strong> 2017, doi: 10.1053/j.gastro.2017.11.275<br />
</li>
<li>Wang Y, Lin X, Chen X, Chen X, Xu Z, Zhang W, Liao Q, <U>Duan X</U>, <strong>Wang X</strong>, Liu M, Wang F, He J &amp; Shi P, Tetherless near-infrared control of brain activity in behaving animals using fully implantable upconversion microdevices, <strong>Biomaterials</strong> 2017, 142, 136-148</li>
</ol>
</div>
<div id="section-2" class="section level4">
<h4>2016</h4>
<ol style="list-style-type: decimal">
<li>Yu VWC, Yusuf RZ, Oki T, Wu J, Saez B, <strong>Wang X</strong>, Cook C, Baryawno N, Ziller MJ, Lee E, Gu H, Meissner A, Lin CP, Kharchenko PV, Scadden DT, Epigenetic Memory Underlies Cell-Autonomous Heterogeneous Behavior of Hematopoietic Stem Cells, <strong>Cell</strong>, 167(5):1310 - 1322</li>
<li>Trinh A, Trumpi K, De Sousa E Melo F, <strong>Wang X</strong>, de Jong JH, Fessler E, Kuppen PJK, Reimers MS, Swets M, Koopman M, Nagtegaal I, Jansen M, Hooijer GKJ, Offerhaus GJ, Kranenburg O, Punt CJ, Medema JP, Markowetz F, Vermeulen L, Practical and Robust Identification of Molecular Subtypes in Colorectal Cancer by Immunohistochemistry, <strong>Clinical Cancer Research</strong> 2016, doi: 10.1158/1078-0432.CCR-16-0680</li>
<li>Fessler E, Drost J, van Hooff SR, Linnekamp JF, <strong>Wang X</strong>, Jansen M, De Sousa E Melo F, Prasetyanti PR, IJspeert JEG, Franitza M, Nürnberg P, van Noesel CJM, Dekker E, Vermeulen L, Clevers H, and Medem JP, TGFβ signaling directs serrated adenomas to the mesenchymal colorectal cancer subtype, <strong>EMBO Molecular Medicine</strong> 2016, 8(7): 745–760, doi: 10.15252/emmm.201606184</li>
<li>Fessler E, Jansen M, De Sousa E Melo F, Zhao L, Prasetyanti PR, Rodermond H, Kandimalla R, Linnekamp JF, Franitza M, van Hooff SR, de Jong JH, Oppeneer SC, van Noesel CJM, Dekker E, Stassi G, <strong>Wang X*</strong>, Medema JP*, and Vermeulen L*, A multidimensional network approach reveals microRNAs as determinants of the mesenchymal colorectal cancer subtype, <strong>Oncogene</strong> 2016, doi: 10.1038/onc.2016.134 (co-corresponding author)</li>
<li>Masuda T, <strong>Wang X</strong>, Maeda M, Canver MC, Sher F, Funnell APW, Fisher C, Suciu M, Martyn GE, Norton LJ, Zhu C, Kurita R, Nakamura Y, Xu J, Higgs DR, Crossley M, Bauer DE, Orkin SH, Kharchenko PV, Maeda T, Transcription factors LRF and BCL11A independently repress expression of fetal hemoglobin, <strong>Science</strong> 2016, 351(6270):285-289</li>
</ol>
</div>
<div id="section-3" class="section level4">
<h4>2015</h4>
<ol style="list-style-type: decimal">
<li>Guinney J<sup>†</sup>, Dienstmann R<sup>†</sup>, <strong>Wang X<sup>†</sup></strong>, de Reyniès A<sup>†</sup>, Schlicker A<sup>†</sup>, Soneson C<sup>†</sup>, Marisa L<sup>†</sup>, Roepman P<sup>†</sup>, Nyamundanda G<sup>†</sup>, Angelino P, Bot BM, Morris JS, Simon IM, Gerster S, Fessler E, De Sousa E Melo F, Missiaglia E, Ramay H, Barras D, Homicsko K, Maru D, Manyam GC, Broom B, Boige V, Perez-Villamil B, Laderas T, Salazar R, Gray JW, Hanahan D, Tabernero J, Bernards R, Friend SH, Laurent-Puig P, Medema JP, Sadanandam A, Wessels L, Delorenzi M, Kopetz S, Vermeulen L &amp; Sabine Tejpar, The consensus molecular subtypes of colorectal cancer, <strong>Nature Medicine</strong> 2015, 10.1038/nm.3967</li>
<li>Alekseyenko AA<sup>†</sup>, Walsh EM<sup>†</sup>, <strong>Wang X<sup>†</sup></strong>, Grayson AR, Hsi PT, Kharchenko PV, Kuroda MI, and French CA, The oncogenic BRD4-NUT chromatin regulator drives aberrant transcription within large topological domains, <strong>Genes Dev</strong> 2015, 29: 1507-1523</li>
<li><strong>Wang X</strong> and Markowetz F, “Joining the dots–network analysis of gene perturbation data”, book chapter, In Systems Genetics: linking phenotype and genotype, Markowetz F and Boutros M, Cambridge University Press, 2015</li>
</ol>
</div>
<div id="section-4" class="section level4">
<h4>2014</h4>
<ol style="list-style-type: decimal">
<li>Sadanandam A<sup>†</sup>, <strong>Wang X<sup>†</sup></strong>, Felipe de Sousa E, Gray JW, Vermeulen L, Hanahan D, &amp; Medema JP, Reconciliation of classification systems defining molecular subtypes of colorectal cancer, <strong>Cell Cycle</strong> 2014, 13(3): 353-357</li>
<li><strong>Wang X</strong>, Yuan K, Hellmayr C, Liu W and Markowetz F, Reconstructing evolving signaling networks by hidden Markov nested effects models, <strong>Annals of Applied Statistics</strong> 2014, 8(1): 448-480</li>
</ol>
</div>
<div id="section-5" class="section level4">
<h4>2013</h4>
<ol style="list-style-type: decimal">
<li>De Sousa E Mello, F<sup>†</sup>, <strong>Wang X<sup>†</sup></strong>, Jansen M, Fessler E, Trinh A, et al., Poor prognosis colon cancer is defined by a distinct molecular subtype and develops from serrated precursor lesions, <strong>Nature Medicine</strong> 2013, 19(5): 614-618</li>
</ol>
</div>
<div id="section-6" class="section level4">
<h4>~2012</h4>
<ol style="list-style-type: decimal">
<li>Mulder KW, <strong>Wang X</strong>, Escriu C, Ito Y, Schwarz RF, Gillis J, et al., Diverse epigenetic strategies interact to control epidermal differentiation. <strong>Nature Cell Biology</strong> 2012, 14(7):753-763</li>
<li><strong>Wang X</strong>, Castro MA, Mulder KW and Markowetz F, Posterior association networks and functional modules inferred from rich phenotypes of gene perturbations. <strong>PLoS Computational Biology</strong> 2012, 8(6):e1002566</li>
<li>Castro MA, <strong>Wang X</strong>, Fletcher M, Ponder BAJ, Meyer KB, Markowetz F, RedeR: R/Bioconductor package for representing modular structures, nested networks and multiple levels of hierarchical associations, <strong>Genome Biology</strong> 2012, 13:R29</li>
<li><strong>Wang X<sup>†</sup></strong>, Terfve C<sup>†</sup>, Rose JC, Markowetz F, HTSanalyzeR: an R/Bioconductor package for integrated network analysis of high-throughput screens. <strong>Bioinformatics</strong> 2011, 27(6):879</li>
</ol>
</div>
</div>

<p>Copyright &copy; 2016-2018 Feng GAO All rights reserved.</p>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
